frontend/src/lib/components/PropertyFilters/PropertyValue.js

```diff
@@ -1,66 +1,56 @@
 import React, { Component } from 'react'
 import api from '../../api'
-import AsyncCreatableSelect from 'react-select/async-creatable/dist/react-select.esm'
-import { selectStyle, debounce } from '../../utils'
+import { Select } from 'antd'
+import { debounce } from '../../utils'
 import PropTypes from 'prop-types'
 
 export class PropertyValue extends Component {
     constructor(props) {
         super(props)
-        this.state = { input: props.value }
+        this.state = {
+            input: '',
+            optionsCache: [],
+            options: [],
+        }
         this.loadPropertyValues = debounce(this.loadPropertyValues.bind(this), 250)
-        this.ref = React.createRef()
+        this.loadPropertyValues('')
     }
-    loadPropertyValues(value, callback) {
+    loadPropertyValues(value) {
         let key = this.props.propertyKey.split('__')[0]
-        api.get('api/' + this.props.endpoint + '/values/?key=' + key + (value ? '&value=' + value : '')).then(
-            propValues =>
-                callback(
-                    propValues.map(property => ({
-                        label: property.name ? property.name : '(empty)',
-                        value: property.name,
-                    }))
-                )
+        this.setState({ optionsCache: { ...this.state.optionsCache, [value]: true } })
+        api.get('api/' + this.props.type + '/values/?key=' + key + (value ? '&value=' + value : '')).then(propValues =>
+            this.setState({
+                options: [...new Set([...this.state.options, ...propValues.map(option => option.name)])],
+            })
         )
     }
     render() {
-        let { propertyKey, onSet, value } = this.props
-        let { isEditing, input } = this.state
+        let { onSet, value } = this.props
+        let { input, optionsCache, options } = this.state
+        options = options.filter(option => input === '' || option.toLowerCase().indexOf(input.toLowerCase()) > -1)
```

Comment:
need object handling here. trying to filter by name will still cause page to crash. name options are still in object form {last, first}